README.md

```diff
@@ -277,6 +277,23 @@ Sometimes you may need to hide a certain route from the documentation, there is
 - Pass `{ hide: true }` to the schema object inside the route declaration.
 - Use the tag declared in `hiddenTag` options property inside the route declaration. Default is `X-HIDDEN`.
 
+<a name="integration"></a>
+### Integration
+You can integration this plugin with ```fastify-helmet``` with some little work.
+
+```fastify-helmet``` options example:
+```javascript
+.register(helmet, {
+  contentSecurityPolicy: {
+    directives: {
+      imgSrc: ["'self'", 'data:', 'validator.swagger.io'],
+      scriptSrc: ["'self'"].concat(fastify.swaggerCSP.script),
+      styleSrc: ["'self'", 'https:'].concat(fastify.swaggerCSP.style)
+    }
+  }
+})
```

Comment:
Can you please write a test that verify this works correctly?

Reply:
I have created the integration test and update the ```REMEAD.md``` according to the testing.
I do not know if it is the right way for integration test, but it works.